controls/templatedbutton/visual-states.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Visual States
+page_title: .NET MAUI TemplatedButton Documentation - Visual States
+description: Learn how to set the border color, border thickness and other in different Visual States of the Telerik TemplatedButton for .NET MAUI.
+position: 9
+tags: .net maui, telerik .net maui, ui for .net maui, templated, button, microsoft .net maui
+slug: templatedbutton-visual-states
+---
+
+# .NET MAUI TemplatedButton Visual States
+
+The TemplatedButton provides the following visual states described in the table below:
```

Comment:
```suggestion
The TemplatedButton provides the following `CommonStates` visual states:
```

Reply:
Btw, doesn't TemplatedButton have focus states as well?

Reply:
no, only applicable for togglebutton